app/core/initialize.js

```diff
@@ -43,30 +43,39 @@ if (utils.isOzaria) {
 }
 
 if(utils.shaTag !== 'dev') { // tracking only in production
-  datadogRum.init({
-    ...(utils.isCodeCombat ? {
-      applicationId: '0fe8c7ec-5984-4191-b5f3-666fa6617477',
-      clientToken: 'pubb0ab7e396292808398f9cccbd7cdde42',
-      service: 'coco-client'
-    } : {
-      applicationId: 'be527613-fee9-4a9d-a0b7-7bf28c35946f',
-      clientToken: 'pub1d3aa1bffdc982bc39deba66302a4f31',
-      service: 'ozaria-client'
-    }),
-    site: 'datadoghq.com',
-    env: 'production',
-    version: utils.shaTag, // we can use it as version for now. or do we have a version somewhere?
-    sessionSampleRate: 100,
-    sessionReplaySampleRate: 20,
-    trackUserInteractions: true,
-    trackResources: true,
-    trackLongTasks: true,
-    defaultPrivacyLevel: 'mask-user-input',
-    allowedTracingUrls: [
-      /https:\/\/.*\.codecombat\.com/,
-      /https:\/\/.*\.ozaria\.com/,
-    ]
-  })
+  const DD_RUM_RANDOM_NUMBER_KEY = 'ddRumRandomNumber'
```

Comment:
good to add userId in the key so that we don't use same key across different users using same browser.

Reply:
@mrfinch we don't have the `me` here yet :-/ But since it is `sessionStorage` and not `localStorage`, if they close+reopen the browser, or even if they open up a new tab, it'll be a new session.